velox/connectors/hive/tests/HiveDataSinkTest.cpp

```diff
@@ -505,6 +507,42 @@ TEST_F(HiveDataSinkTest, basic) {
   verifyWrittenData(outputDirectory->path);
 }
 
+TEST_F(HiveDataSinkTest, parquetBlockSizeAndRows) {
+  std::unordered_map<std::string, std::string> connectorConfig;
+  connectorConfig.emplace("parquet_optimized_writer_max_block_size", "10L");
```

Comment:
Does this mean the max Row Group size is 10 bytes?

Reply:
Yes.